ride-service/src/main/java/com/modsen/rideservice/services/RideServiceImpl.java

```diff
@@ -0,0 +1,164 @@
+package com.modsen.rideservice.services;
+
+import com.modsen.rideservice.dto.RideRequest;
+import com.modsen.rideservice.dto.RideResponse;
+import com.modsen.rideservice.dto.RideResponseList;
+import com.modsen.rideservice.entities.Ride;
+import com.modsen.rideservice.exceptions.*;
+import com.modsen.rideservice.mappers.RideMapper;
+import com.modsen.rideservice.repositories.RideRepository;
+import com.modsen.rideservice.services.interfaces.RideService;
+import com.modsen.rideservice.util.ExceptionMessages;
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Service;
+
+import java.time.LocalDate;
+import java.util.Objects;
+import java.util.stream.Collectors;
+
+@Service
+@RequiredArgsConstructor
+public class RideServiceImpl implements RideService {
+
+    private final RideRepository repository;
+
+    private final RideMapper mapper;
+
+    public RideResponseList getAll()
+    {
+        return new RideResponseList(repository.findAll().stream()
+                .map(mapper::entityToResponse)
+                .collect(Collectors.toList()));
+    }
```

Comment:
На стажировке мы придерживаемся другого code-style:
```suggestion
    public RideResponseList getAll() {
        return new RideResponseList(repository.findAll().stream()
                .map(mapper::entityToResponse)
                .collect(Collectors.toList()));
    }
```
Поправь в остальных местах под этот